legal-api/tests/unit/resources/v2/test_business_filings/test_filings.py

```diff
@@ -893,12 +894,7 @@ def test_update_ar_with_a_missing_filing_id_fails(session, client, jwt):
                                 last_ar_date=datetime(datetime.utcnow().year - 1, 4, 20).date()
                                 )
     factory_business_mailing_address(business)
-    ar = copy.deepcopy(ANNUAL_REPORT)
-    annual_report_date = datetime(datetime.utcnow().year, 2, 20).date()
-    if annual_report_date > datetime.utcnow().date():
-        annual_report_date = datetime.utcnow().date()
-    ar['filing']['annualReport']['annualReportDate'] = annual_report_date.isoformat()
-    ar['filing']['annualReport']['annualGeneralMeetingDate'] = datetime.utcnow().date().isoformat()
+    ar = copy.deepcopy(REGISTRARS_ORDER_FILING_TEMPLATE)
```

Comment:
why are we changing this test?  what problem did you run into with the original implementation?